types/api.d.ts

```diff
@@ -47,10 +59,26 @@ interface RuntimeOptions {
    * @default false
    */
   removeHTMLComments?: boolean;
-  
+
   customParser?: SourceParser;
 }
 
+interface ASTOptions {
```

Comment:
The terminology is misleading, these are not options for the Abstract Syntax Tree ([AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree)). Essentially, an AST is a high-level representation of the source code's syntax, which we traverse in order to identify security vulnerabilities.

Reply:
I have tried to rework a little the type for the options (and the naming), not sure it's 100% correct but lmk if its better like this